actions/install-automation-components/action.yml

```diff
@@ -12,6 +12,8 @@ runs:
           echo "os=ubuntu" | tee -a "$GITHUB_OUTPUT"
         elif command -v microdnf &>/dev/null; then
           echo "os=ubi-minimal" | tee -a "$GITHUB_OUTPUT"
+        elif command -v dnf &>/dev/null; then
+          echo "os=rhel" | tee -a "$GITHUB_OUTPUT"
```

Comment:
This is also the case for `ubi` images (not ubi-minimal). Perhaps we should consider consolidating these in a single step?